projectmanager/RubyRunConfiguration.h

```diff
@@ -0,0 +1,58 @@
+#ifndef RUBYRUNCONFIGURATION_H
+#define RUBYRUNCONFIGURATION_H
+
+#include <projectexplorer/runconfiguration.h>
+
+#include <QWidget>
+
+namespace ProjectExplorer {
+class WorkingDirectoryAspect;
+class ArgumentsAspect;
+class TerminalAspect;
+class LocalEnvironmentAspect;
+}
+
+namespace Ruby {
+
+class RubyRunConfiguration : public ProjectExplorer::RunConfiguration
```

Comment:
Could you just rename the class from `Ruby::RubyRunConfiguration` to `Ruby::RunConfiguration`?